src/common/components/select/Select.module.css

```diff
@@ -1,56 +1,61 @@
 .select {
-    display: flex;
-    flex-direction: column;
-    width: 100%;
+  display: flex;
```

Comment:
Apologies for the whitespace changes here. I used my IDE's replace tool so I did not notice that whitespace had been changed.